templates/scripts/sync-up-version.js

```diff
@@ -0,0 +1,42 @@
+const fs = require("fs");
+var path = require("path");
+const loadJsonFile = require("load-json-file");
+const writePkg = require("write-pkg");
+const semver = require("semver");
+
+const sdk_version = require(path.join(__dirname, "../../packages/sdk/package.json")).version;
+const sdk_name = require(path.join(__dirname, "../../packages/sdk/package.json")).name;
+
+function listFile(dir, list = []) {
+  var arr = fs.readdirSync(dir);
+  arr.forEach(function (item) {
+    var fullpath = path.join(dir, item);
+    var stats = fs.statSync(fullpath);
+    if (item === "node_modules") return list;
+    if (stats.isDirectory()) {
+      listFile(fullpath, list);
+    } else {
```

Comment:
I prefer `else if () {}` instead of `else { if () {} }`

Reply:
sure